src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php

```diff
@@ -67,6 +67,7 @@ class UnusedTagsPass implements CompilerPassInterface
         'routing.expression_language_provider',
         'routing.loader',
         'routing.route_loader',
+        'security.authenticator_manager',
```

Comment:
Just asking: could "authenticator manager" be renamed as "authenticator" everywhere?

Reply:
You're catching exactly one of the remaining pain points in this PR :)

The term "manager" is too broad and not nice. However, the term "Authenticators" is now used for each specific authentication mechanism (i.e. form login authenticator, remember me authenticator, etc). This is in line with "guard authenticators".

So I think calling the class that calls all authenticators an authenticator is confusing. Yet I do agree that we have to have a small brainstorm about a better name for this class than a "manager".